### PR TITLE
Identity-2952 add jit_enabled to organization CLI

### PR DESCRIPTION
**Pull request by LucasLyld**
Release Notes
-------------

Breaking Changes
- PLACEHOLDER

New Features
- Add `--jit-enabled` to allow customer to toggle the JUST-IN-TIME for SSO enabled organization

Bug Fixes
- PLACEHOLDER

Checklist
---------
1. [CRUCIAL] Is the change for features that are already live in prod?
   * yes: ok
   * no: DO NOT MERGE until the required features are live in prod  

What
----

References
----------

Test & Review
-------------

Open Questions / Follow-ups
---------------------------

### CODE REVIEW

File: internal/organization/command_describe.go [hunk 1]
```diff
@@ -10,9 +10,10 @@ import (
 )
 
 type out struct {
-	IsCurrent bool   `human:"Current" serialized:"is_current"`
-	Id        string `human:"ID" serialized:"id"`
-	Name      string `human:"Name" serialized:"name"`
+	IsCurrent   bool   `human:"Current" serialized:"is_current"`
+	Id          string `human:"ID" serialized:"id"`
+	Name        string `human:"Name" serialized:"name"`
+	JIT_Enabled bool   `human:"JIT_Enabled" serialized:"jit_enabled"`
```
Comment (sgagniere) [inline internal/organization/command_describe.go:16]:
```suggestion
	JitEnabled bool   `human:"JIT Enabled" serialized:"jit_enabled"`
```
We generally avoid underscores in our names. The human output shouldn't have underscores either.

File: internal/organization/command_update.go [hunk 2]
```diff
@@ -20,30 +20,41 @@ func (c *command) newUpdateCommand() *cobra.Command {
 	}
 
 	cmd.Flags().String("name", "", "Name of the Confluent Cloud organization.")
+	cmd.Flags().Bool("jit-enabled", false, "The flag to toggle Just-In-Time user provisioning for SSO-enabled organization.")
```
Comment (sgagniere) [inline internal/organization/command_update.go:23]: Is this a one time operation, or can users disable it after enabling it? I think we can reword the flag description based depending on how this works.

Comment (LucasLyld) [reply internal/organization/command_update.go:23]: the latter. the description is consistent with the public api one

Comment (brianstrauch) [reply internal/organization/command_update.go:23]:
```suggestion
	cmd.Flags().Bool("jit-enabled", false, "Toggle Just-In-Time (JIT) user provisioning for SSO-enabled organizations.")
```

**Comment by brianstrauch** [comment]
@LucasLyld does this flag need to be hidden behind a feature flag, or will the feature be GA when this PR is merged?

File: internal/organization/command_describe.go (hunk 1 above)
Comment (LucasLyld) [reply internal/organization/command_describe.go]: updated

File: test/test-server/org_handlers.go [hunk 3]
```diff
@@ -108,9 +109,9 @@ func handleOrgOrganizations(t *testing.T) http.HandlerFunc {
 		switch r.Method {
 		case http.MethodGet:
 			organizationList := &orgv2.OrgV2OrganizationList{Data: []orgv2.OrgV2Organization{
-				{Id: orgv2.PtrString("abc-123"), DisplayName: orgv2.PtrString("org1")},
-				{Id: orgv2.PtrString("abc-456"), DisplayName: orgv2.PtrString("org2")},
-				{Id: orgv2.PtrString("abc-789"), DisplayName: orgv2.PtrString("org3")},
+				{Id: orgv2.PtrString("abc-123"), DisplayName: orgv2.PtrString("org1"), JitEnabled: orgv2.PtrBool(true)},
+				{Id: orgv2.PtrString("abc-456"), DisplayName: orgv2.PtrString("org2"), JitEnabled: orgv2.PtrBool(true)},
+				{Id: orgv2.PtrString("abc-789"), DisplayName: orgv2.PtrString("org3"), JitEnabled: orgv2.PtrBool(true)},
```
Comment (brianstrauch) [inline test/test-server/org_handlers.go:114]: Let's make one of these `false`, just for some variation.
```suggestion
				{Id: orgv2.PtrString("abc-789"), DisplayName: orgv2.PtrString("org3"), JitEnabled: orgv2.PtrBool(false)},
```

**Review by brianstrauch — APPROVED** [review]

**Comment by brianstrauch** [comment]
> it doesn't need. Currently it's required for LA, and only certain groups of orgs which is already onboard can use this feature flag, we would return proper error message to tell customer to onboard to AUPM first

Got it, so for users who haven't been onboarded, they'll see `jit_enabled: false` when they use the `describe` command, and see a backend error message if they use the `--jit-enabled` flag during `update`.

**Review by sgagniere — APPROVED** [review]

**Pull request merged**